gc/CMakeLists.txt

```diff
@@ -167,6 +167,9 @@ add_library(omrgc STATIC
 	stats/ScavengerStats.cpp # TODO only compile if scavenger or VLHGC. Is this actually used by VLHGC?
 	stats/SweepStats.cpp
 
+	stats/ConcurrentGCStats.cpp
+	stats/CompactStats.cpp
```

Comment:
Why did these have to move out of the proper location for their compile flags?

Reply:
IIRC The stats aren't ifdef'd properly, this was giving me a compile error when compact or concurrent were disabled.